### DISCUSSION TITLE
Rather noob question RE logging in

### DISCUSSION THREAD

**Discussion by threemoons-nyc** [mastodon/mastodon#22370] [Q&A] [answered]
Sounds dumb, but what if I forget which Mastadon server I'm on?  Is there a place where I can pop an email and have it automagically remind me so that I am logging in to the correct server?  Thanks!

**Comment by mohe2015** [comment]
I don't think there is. Two options that come to my mind is to either find your registration email or to use the search on a popular instance to maybe find your account. Though I'm not sure how the search works so maybe that's not possible either.

**Comment by nightpool** [comment, marked as answer]
@threemoons-nyc No, there's no way to publicly look up a user by email address—it would be a huge privacy violation to be able to share user's private email addresses like that.

I would recommend looking through your email inbox for records of your registration

**Reply by threemoons-nyc** [reply]
Oh, I didn't mean  looking up a user by email.  I mean, I wish that there was a way that Mastadon would let you pop in an email that you own and email you back your server.  If such a tool doesn't exist yet, maybe it should?  Thanks!

**Reply by mohe2015** [reply]
> Oh, I didn't mean looking up a user by email. I mean, I wish that there was a way that Mastadon would let you pop in an email that you own and email you back your server. If such a tool doesn't exist yet, maybe it should? Thanks!

Unfortunately it's still technically impossible as that would mean there would need to be a central place that stores all email -> account associations.

**Reply by nightpool** [reply]
@threemoons-nyc Yes, from a technical perspective, one would require the other.

**Reply by mohe2015** [reply]
@threemoons-nyc
Though it looks like your twitter profile contains your mastodon account handle so it shouldn't be too hard. (the part behind the second @ symbol is the website you registered on in case you're looking for that account)

**Reply by threemoons-nyc** [reply]
Thanks, yeah, put my handle in my Twitter handle so that people could find me.  And, now, I can also find myself (until Elon suspends me).  Thank you!  :)
